sql/api/src/test/scala/org/apache/spark/sql/RowSuite.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql
+
+import org.apache.spark.{SparkException, SparkFunSuite}
+import org.apache.spark.sql.catalyst.expressions.GenericRow
+
+class RowSuite extends SparkFunSuite {
+  test("Row handles null value") {
```

Comment:
Can you move it to some existing test suite?

Reply:
`sql/api` doesn't have any other tests. This was adding the first one. 

Do you have a different package you want me to move it to?

Additionally, I think the dependency above is required in `test` because `SparkFunSuite` lives in `core`. I am not putting a dependency for the main code on `core`, just the test code. I'm fine changing that, but would need more guidance as to where I should put the test